README.md

```diff
@@ -79,12 +79,33 @@ Example:
 cti validate
 ```
 
-##### cti pack
+### cti pack
 
-Packs the package into a bundle.
+Packs the package into a bundle. Valid package should be in current working directory (or directory specified by `--working-dir`).
```

Comment:
Current working directory and global flag is already mentioned in the note above. I think it's not necessary here.